server/index.js

```diff
@@ -1,3 +1,9 @@
+/* 
+**  @Deprecation notice
```

Comment:
I actually wouldn't want to deprecate this - my intention is to keep it solely for development. In production we use the CLI as discussed.